src/game_view.h

```diff
@@ -31,13 +31,18 @@ class game_view
   /// Get const reference to m_game_options
   const game_options& get_options() const noexcept {return get_game().get_game_options();}
 
+  void set_options(const game_options& new_options) noexcept { m_game.set_options(new_options); }
+
   /// Processes events in game and ouputs false if quit
   /// is inputted
   bool process_events();
 
   ///Gets a const ref to m_game
   const game& get_game() const noexcept {return m_game; }
 
+  ///Gets a ref to m_game
+  game& get_game() noexcept {return m_game; }
```

Comment:
Ooooh, this is scary, as one should [C.9: Minimize exposure of member](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-private): making `game` changable by ref is quite of the opposite. However, I just mention this so you can make a mental note and then observe the havoc this may cause. And hey, maybe I am wrong :innocent:

Reply:
Rm'd it, but in my defence, I didn't add this to the game, only changed its place 😇 
It seems that it was used for profiling.

Reply:
Ah, like that :-) . Thanks for the deletion then :grin: